larsim/MergeSimSources/MergeSimSources_module.cc

```diff
@@ -360,6 +407,14 @@ void sim::MergeSimSources::produce(art::Event & e)
     } // for
   } // if fill energy deposits
 
+  if(fFillAuxDetHits) {
+    for (auto&& [ auxdethit_inst, auxdethitCol ]
+      : util::zip(fAuxDetHitsInstanceLabels, auxdethitCols))
+    {
+      e.put(std::move(auxdethitCol), auxdethit_inst);
```

Comment:
... I suggest
```c++
e.put(std::make_unique<aux_det_hits_t>(move(auxdethitCol)), auxdethit_inst);
```
Should provide similar efficiency.

Reply:
Done with commit db484f9